seekinternational/seek-asia-style-guide
# fix(JobCard): List disc doesn't align in IE.

We had a bug in IE, where the list disc wasn't aligning to the content properly. mainly as the internal `<li />` > `<Text/>` was throwing things out of whack.

The ability for us to mess with list-styles are pretty limited and not consistent within browsers, so the change has been made to assigning a `::before` element to the `li` with the disc doodad.

Before:

![screen shot 2019-02-14 at 3 28 16 pm](https://user-images.githubusercontent.com/19305834/52763244-20204680-306f-11e9-86b3-be624207e648.png)


After:

![screen shot 2019-02-14 at 3 28 30 pm](https://user-images.githubusercontent.com/19305834/52763253-24e4fa80-306f-11e9-9790-a58591ed0c45.png)

To Test:

Cross browser check that everything looks good. Especially in IE.

The rule of thumb I've gone off is that the disc should now align with the rest of the location and price icons below, and the text should do the same.

Make sure to test list items with multiple lines of text, so that the disc icon always keeps aligned to the top.
